tests/cypress/e2e/components/media.cy.ts

```diff
@@ -98,7 +100,38 @@ describe('Media component', () => {
   mediaType.forEach(
     ({ url, title, description, collection, creator, tagRelated, type }) => {
       it(`should render ${type} in Media component`, () => {
+        const isRmrk = url.startsWith('/rmrk')
+
+        if (isRmrk) {
+          cy.intercept('POST', '**/rubick/007/graphql', (req) => {
+            // Queries
+            aliasQuery(req, 'nftById')
+            aliasQuery(req, 'collectionById')
+          })
+        } else {
+          cy.intercept('POST', '**/snekk/003/graphql', (req) => {
+            // Queries
+            aliasQuery(req, 'nftById')
+            aliasQuery(req, 'collectionById')
+          })
+        }
+
         cy.visit(url)
+
+        cy.intercept('GET', '**/kodadot.mypinata.cloud/ipfs/**').as(
+          'fetchMedia'
+        )
+
+        // wait for graphql request to be fetched
+        cy.wait('@nftByIdQuery', { responseTimeout: 15000 })
+          .its('response.statusCode')
+          .should('match', /^(200|206)/)
+
+        // wait for asset to be loaded from pinata
+        cy.wait('@fetchMedia', { responseTimeout: 20000 })
+          .its('response.statusCode')
+          .should('match', /^(200|206)/)
```

Comment:
hi @kkukelka how about like this
```js
cy.wait('@fetchMedia', { responseTimeout: 20000 }).its('response.statusCode').then(code => {
  if (code === 200 || code === 206) {
    doTheTest()
  }

  // else ignore the test
})
```

Reply:
kind of tricky to do in this case since i want/need to test if the data streams of video and audio work correctly. in these cases there will be multiple fetchMedia wait assertions which i now limited to 5

Reply:
and i'm still having trouble with audio for now... somehow when not running cypress in the GUI it just isnt abled to load the audio stream